extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -665,6 +665,11 @@ export class Carousel {
       this.scrollContainer_.setAttribute('horizontal', this.axis_ === Axis.X);
       this.scrollContainer_.setAttribute('loop', this.isLooping());
```

Comment:
It looks like we set `loop` value on the scroll container. Can we, instead of setting `loop` on the custom element, instead modify our CSS rules to look for the correct loop value on this inner element?

Reply:
Yes, but the CSS rule would have to be very specific, which AFAIK is ok since we handle the creation of the arrows for all amp-base-carousel components and friends. 

Does that sounds right to you?